### PR TITLE
Login page

### PR DESCRIPTION
**Pull request by gaetan-cordonnier**

### CODE REVIEW

**Review by AmjadLille — DISMISSED** [review]

**Review by LoicBrassart — DISMISSED** [review]
SnE

**Review by AmjadLille — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -57,6 +57,5 @@
   "devDependencies": {
     "concurrently": "^5.3.0",
     "prettier": "^2.2.1"
-  },
-  "proxy": "http://localhost:5000"
+  }
```
Comment (LoicBrassart) [inline package.json:60]: ... Still doubting

**Review by LoicBrassart — CHANGES_REQUESTED** [review]
I want to see if these PR break `dev`

**Review by LoicBrassart — APPROVED** [review]
Nope, that's me, sorry!

**Pull request merged**